### ISSUE TITLE
With cryptobiolin/subjugate, you can move diagonally

### ISSUE THREAD

**Issue by 9600bauds** [d3athrow/vgstation13#12914]
feachur??

**Comment by Exxion** [comment]
How do you mean?

**Comment by 9600bauds** [comment]
You know when you move randomly with cryptobiolin? Just hold down a key and you will randomly move diagonally, like how ghosts can move now. I really don't think this lets you move through diagonal gaps but I haven't tested it.

**Comment by Exxion** [comment]
I actually know nothing at all about cryptobiolin

If this is NEW behavior, you probably can't move through diagonal gaps with it, but if this existed before the moveloop update, you probably can

**Comment by 9600bauds** [comment]
Nope, can't escape
![dreamseeker_2016-12-20_15-35-04](https://cloud.githubusercontent.com/assets/6526157/21363094/e8b4f21a-c6c9-11e6-94e8-344bde8fcc28.png)

So it's mostly just the person looking weird as they somehow move diagonally while they bump into every wall in the room

**Comment by Sprok0** [comment]
It's anything that applies the "confused" effect, it makes you walk in random directions. For some reason it can also select from the diagonal directions too.

**Comment by 9600bauds** [comment]
I think more specifically it's the recent movement-keypress-queueing thing that Clusterfack was trying to restrict diagonal movements from

**Comment by Sprok0** [comment]
No, this has been the behaviour forever.
```
		else if(mob.confused)
			step_rand(mob)
			mob.last_movement=world.time
```
https://github.com/d3athrow/vgstation13/blob/Bleeding-Edge/code/modules/mob/mob_movement.dm
Line 371

Mechs do the same thing when their actuator or whatever is broken

**Comment by Exxion** [comment]
Obviously, I have more experience with people being drunk than with any other thing that makes people move uncontrollably, but at least with drunkenness, it has always been that you could only move in cardinal directions when free-standing, but you could step diagonally around things.
I have no idea if that's related, though

**Comment by Probe1** [comment]
This isn't the first or last thing to achieve this and it's a top level strategy.
